jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+<robot name="fetch" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:include filename="/etc/ros/indigo/calibrated_latest.urdf" />
```

Comment:
@708yamaguchi we include this file, and we made symbolic link for the latest calibrated urdf.